.github/workflows/rotki_dev_builds.yml

```diff
@@ -58,6 +58,10 @@ jobs:
           key: ${{ runner.os }}-pnpm-store-${{ hashFiles('**/pnpm-lock.yaml') }}
           restore-keys: |
             ${{ runner.os }}-pnpm-store-
+      - name: Setup rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
```

Comment:
@yabirgb can you find an alternative to this action? 

I checked the warnings about set-output/node12 in the logs and then went to the repo, and the last update was a bit more than 3 years ago, which makes it unmaintained. This essentially means that in the near future it will break. 

If there is no official action, we should do the setup by a script.